packages/safe-core-sdk/src/contracts/safeDeploymentContracts.ts

```diff
@@ -7,8 +13,32 @@ import {
   getSafeSingletonDeployment,
   SingletonDeployment
 } from '@gnosis.pm/safe-deployments'
+import { ContractNetworkConfig } from '../types'
 import { safeDeploymentsL1ChainIds, safeDeploymentsVersions } from './config'
 
+interface GetSafeContractInstanceProps {
```

Comment:
There seems to be some repetition in these types? Maybe it's worthwhile making a base `ContractInstanceProps` type?